src/validate.test.js

```diff
@@ -0,0 +1,138 @@
+/** @jest-environment jsdom */
+
+import Display from './to-do.js';
+import { check, clearCompleted } from './validate.js';
+
+describe('Edit task', () => {
+  beforeEach(() => {
+    localStorage.clear();
+    jest.clearAllMocks();
+    localStorage.setItem.mockClear();
+    document.body.innerHTML = `
+      <div class="todo-body"></div>
+      <div class="todo-edit flex">
+                <input
+                  type="text"
+                  class="no-outline type-task"
+                  placeholder="Add to your list..."
+                  value="Visit the Gym" 
+                />
+                <i class="enter-btn fa-solid fa-right-from-bracket"></i>
+              </div>
+  `;
+    const obj1 = {
+      text: 'Visit the gym with Andre',
+      completed: false,
+      index: 1,
+    };
+    const obj2 = {
+      text: 'Visit the gym with Trina',
+      completed: false,
+      index: 2,
+    };
+    window.localStorage.setItem('todo', JSON.stringify([obj1, obj2]));
+  });
+
+  test('Edit todo object in local storage', () => {
+    const editTodoOnSpy = jest.spyOn(Display, 'editTodo');
+    Display.editTodo(0);
+    expect(editTodoOnSpy).toHaveBeenCalledTimes(1);
+    const result = JSON.parse(window.localStorage.getItem('todo'));
+    expect(result.length).toBe(2);
+    expect(result[0].text).toBe('Visit the gym with Andre');
+  });
+});
```

Comment:
- [ ] The test for editing tasks was not properly written. You are testing for the task that was inputted and not a task that was edited. To successfully test the function for editing a task, you need to change the description for a task and check if it returns the edited task. Kindly implement this change 👍 💻